packages/blogs-client/src/templates/displayPage/index.tsx

```diff
@@ -75,8 +75,8 @@ const DisplayPage: React.FC<Props> = ({ pageContext: { blog } }: Props) => {
           crossOrigin="anonymous"
         />
       </Helmet>
+      <img className="flex-auto h-64 w-full border-solid" src={blog.thumbnailURL} alt={blog.name} />
```

Comment:
is this intended or a mistake?

Reply:
@chantalchawki intended to follow the wireframes, the div has max-w-screen-md while the thumbnail should be full width